frontends/user-articles/UserArticlesSidebar.tsx

```diff
@@ -62,12 +63,27 @@ export const UserArticlesSidebar: FC = () => {
     if (!editor || !article) return Promise.reject();
 
     const editorData = await editor?.save();
-    const blocks = removeAmazonUriFromImgBlocks(editorData.blocks);
-    const title = blocks.find((block) => block.type === 'header')?.data.text;
 
-    const updatedArticle = await updateArticle({ ...article, title, blocks }).unwrap();
+    // Don't save image urls in database. Only image IDs
+    const oldBlocks = removeAmazonUriFromImgBlocks(editorData.blocks);
+    const title = oldBlocks.find((block) => block.type === 'header')?.data.text;
+
+    const updatedArticle = await updateArticle({ ...article, title, blocks: oldBlocks }).unwrap();
     dispatch(setArticle({ ...article, ...updatedArticle }));
     setAlert('');
+
+    for (let i = 0; i < updatedArticle.blocks.length; i++) {
```

Comment:
old blocks should be looped and if base64 image excists in old blocks render new blocks

Reply:
@samandar-boymurodov and it should be checked for image tool and unsplash tool